src/components/TodoLine/TodoLine.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import classNames from 'classnames';
+
+type Props = {
+  TodoElement: Todo;
```

Comment:
```suggestion
  todo: Todo;
```
Names should be in `camelCase`, in PascalCase only component names and classes